vendor/github.com/elastic/go-libaudit/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Change Log
```

Comment:
Can we put this inside the kernel module vendor directory as we do for other modules?